cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -92,8 +92,8 @@ def __init__(self, scope: core.Construct, construct_id: str, imagebuild: ImageBu
         # InfrastructureConfiguration
         imagebuilder.CfnInfrastructureConfiguration(
             self,
-            id="PClusterImageInfrastructureConfiguration",
-            name="-".join(["PCluster-Image-Infrastructure-Configuration", resources_prefix]),
+            id="ParallelClusterImageInfrastructureConfiguration",
```

Comment:
let's remove the "image" substring. i.e `ParallelClusterInfrastructureConfiguration`

Reply:
done